python/py-pyside2/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem              1.0
 PortGroup               python 1.0
 
 name                    py-pyside2
-version                 5.15.2
+version                 5.15.2.1
 revision                1
```

Comment:
Revision should be reset to 0 (although I personally object to having to do so: if you forgot to reset the revision before building it, you and any other testers for a PR now get to reinstall and possibly rebuild it all over again, which I believe is pointless for ports that take a long time to build).

Reply:
this should indeed be set to `0`